ansible/roles/builder/build/tasks/create_sources.yml

```diff
@@ -10,6 +10,22 @@
     path: /root/rpmbuild/SOURCES
     state: directory
 
+- name: install Modern WebUI dependencies
```

Comment:
Can you just add a comment explaining that the 3 steps (install nodejs, call `npm install` and `npm run build`) are done here in order to create the files for the new webui **outside** of the mock chroot? This way the tar.gz file contains the webui files in `install/freeipa-webui/dist` and they can be directly used in the chroot without any need to install Node.